internal/zinc-persist/src/main/scala/sbt/internal/inc/binary/converters/ProtobufReaders.scala

```diff
@@ -146,14 +143,21 @@ final class ProtobufReaders(mapper: ReadMapper) {
     }
   }
 
+  private def fromString(value: String): Option[String] =
```

Comment:
Is indentation off here compared to `fromInt`?

Reply:
What do you mean? They're both indented the same way.

Reply:
Safari plays tricks on me :stuck_out_tongue: